rust_snuba/src/processors/eap_spans.rs

```diff
@@ -82,19 +134,21 @@ fn fnv_1a(input: &[u8]) -> u32 {
 impl From<FromSpanMessage> for EAPSpan {
     fn from(from: FromSpanMessage) -> EAPSpan {
         let mut res = Self {
-            organization_id: from.organization_id,
+            primary_key: PrimaryKey {
```

Comment:
@untitaker I was curious why exactly we added this here? Is it just so that we have some consistency between the mutations and EAP spans schema?